README.md

```diff
@@ -91,7 +91,7 @@ the previous snippet. A summary of all settings follows:
 | adLabel                | string       | Replaces the "Advertisement" text in the ad label. Added for multilingual UI support. |
 | adsRenderingSettings   | object       | JSON object with ads rendering settings as defined in the IMA SDK,Docs(1). |
 | autoPlayAdBreaks       | boolean      | Whether or not to automatically play VMAP or ad rules ad breaks. Defaults,to true. |
-| adWillPlayMuted        | boolean      | Notifies the SDK whether the player intends to start ad while muted. Changing this setting will have no impact on ad playback. Defaults,to false. |
+| **deprecated** adWillPlayMuted        | boolean      | Notifies the SDK whether the player intends to start ad while muted. Changing this setting will have no impact on ad playback. Defaults,to false. |
```

Comment:
This isn't deprecated by IMA, we're just deprecating the setting in the plugin because we added logic to detect this. We don't need folks who implement the plugin to tell us if their ad will play muted or not - we can just check if it will within the plugin.

Reply:
That makes sense. FYI, the auto-detect wasn't working with our implementation, we got a message from Google saying that we weren't setting this when we should be.